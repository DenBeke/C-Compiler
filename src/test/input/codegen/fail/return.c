// No return at end of non-void function

int a() {
    
}

void main() {
    
}