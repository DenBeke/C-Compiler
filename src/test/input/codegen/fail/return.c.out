[WARNING] line 7: Control may reach end of non-void function 'a' without return
mst 0
cup 0 init
init:
ssp 5
mst 0
cup 0 main1
hlt
a0:
ssp 5
retp
main1:
ssp 5
retp
